api/project_member.go

```diff
@@ -7,6 +7,31 @@ import (
 	"github.com/bytebase/bytebase/common"
 )
 
+// ProjectRoleProvider is the role provider for a user in projects.
+type ProjectRoleProvider string
+
+const (
+	// ProjectRoleProviderBytebase is the role provider of a project.
+	ProjectRoleProviderBytebase ProjectRoleProvider = "BYTEBASE"
+	// ProjectRoleProviderGitLabSelfHost is the role provider of a project.
+	ProjectRoleProviderGitLabSelfHost ProjectRoleProvider = "GITLAB_SELF_HOST"
+)
+
+func (e ProjectRoleProvider) String() string {
+	switch e {
+	case ProjectRoleProviderBytebase:
+		return "BYTEBASE"
+	case ProjectRoleProviderGitLabSelfHost:
+		return "GITLAB_SELF_HOST"
+	}
+	return ""
+}
+
+type ProjectRoleProviderPayload struct {
+	VCSRole string `json:"vcsRole"`
+	SyncTs  int64  `json:"syncTs"`
```

Comment:
Should this be named last sync time? And make comments on these public interfaces following go style, otherwise go linter will complain.